coderd/workspaceagentsrpc_internal_test.go

```diff
@@ -236,7 +242,7 @@ func TestAgentConnectionMonitor_SendPings(t *testing.T) {
 	}()
 	fConn.requireEventuallyHasPing(t)
 	cancel()
-	<-done
+	_ = testutil.TryReceive(testCtx, t, done)
```

Comment:
drive by improvement for consistency and so it won't hang the tests if the pinger deadlocks.